data/datavalid.yml

```diff
@@ -158,6 +158,9 @@ schemas:
       - name: investigator_uid
       - name: investigator_rank
       - name: shift_supervisor
+      - name: allegation_desc
+        description: >
+          text description of the allegation
```

Comment:
@rajivsinclair  Should we rename this field to `summary` to keep it consistent with previous work à la CPDP?

Reply:
I think that `allegation_summary` or `allegation_desc` is more appropriate as `summary` seems vague.

Reply:
I think `allegation_desc` or `allegation_summary` is okay here.

Reply:
that's fine then